backend/server.js

```diff
@@ -22,4 +21,57 @@ function authenticateToken(req, res, next) {
   });
 }
 
-app.listen(5000);
+// Create a new post
+app.post("/posts", authenticateToken, async (req, res) => {
+  const { title, content, imageUrls } = req.body;
+
+  try {
+    const newPost = await prisma.post.create({
+      data: {
+        title,
+        content,
+        user: { connect: { username: req.user.name } },
+        images: {
+          create: imageUrls.map((url) => ({ url })),
+        },
+      },
```

Comment:
Does `prisma` handle data sanitization for you? Might be outside the scope of this PR but just wanted to make sure that's something that's being handled

Reply:
I looked it up to check and this what i found:
https://github.com/prisma/prisma/discussions/7635